.changeset/twelve-rabbits-doubt.md

```diff
@@ -0,0 +1,7 @@
+---
+'@guardian/source-foundations': major
+---
+
+- Removed deprecated exported palette colours culture[350] and opinion[300]. These colours can now be found as culture[400] and opinion[400].
```

Comment:
i don't quite follow this - afaict nothing has changed in what's exported from foundations?

Reply:
In figma they do not have the palette tokens for the deprecated colours culture[350] and opinion[300] so I have used this PR to remove them from the design tokens package and source foundations. These tokens have been deprecated for a long time so thought it was about time they were removed.